tests/data/articles/潮流周刊/source.md

```
@@ -0,0 +1,296 @@
+[
+![潮流周刊](https://gw.alicdn.com/imgextra/i2/O1CN01m9YYjS1QBeW5DOm3I_!!6000000001938-2-tps-400-400.png)
```

markdownlint-cli2 (0.17.2)

Spaces inside link text
null

(MD039, no-space-in-links)